synfig-core/src/synfig/valuenode.h

```diff
@@ -417,10 +417,9 @@ class LinkableValueNode : public ValueNode
 	//! Returns a Loose Handle to the Value Node based on the link's name
 	ValueNode::LooseHandle get_link(const String &name)const { return get_link(get_link_index_from_name(name)); }
 	//! Return a full description of the linked ValueNode given by the index
-	String get_description(int index = -1, bool show_exported_name = true)const;
-	//! Return a full description of the linked ValueNode given by the index
-	//! Proper overload of the inherited function
-	String get_description(bool show_exported_name = true)const;
+	String get_link_description(int index, bool show_exported_name = true)const;
+	//! Return a full description of this linkable ValueNode
+	String get_description(bool show_exported_name = true)const override;
```

Comment:
I added `override` here, so `gcc` now warns about other override methods not being tagged as such.
Should I change them too?

Reply:
I think yes, it's better to tag them too.

Reply:
Done.